app/views/projects/versions/list.scala.html

```diff
@@ -90,6 +90,7 @@ <h4 class="panel-title pull-left">@messages("project.versions")</h2>
                                                     <div class="pull-right">
                                                         <span class="date">@prettifyDate(version.createdAt.get)</span>
                                                         <span class="date">@version.humanFileSize</span>
+                                                        <span class="date">@version.downloadCount Downloads</span>
```

Comment:
This will display `1 Downloads` - should be `1 Download`.

Reply:
I think this should be above the date (to the left of date) as it less important to be next to the download button as well.